dvc/command/plots.py

```diff
@@ -178,6 +145,37 @@ def add_parser(subparsers, parent_parser):
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
     plots_diff_parser.add_argument(
+        "--targets",
+        nargs="*",
+        help="Plots file to visualize. Shows all plots by default.",
+    )
+    plots_diff_parser.add_argument(
+        "revisions", nargs="*", default=None, help="Git commits to plot from",
+    )
+    _add_props_arguments(plots_diff_parser)
+    _add_output_arguments(plots_diff_parser)
+    plots_diff_parser.set_defaults(func=CmdPlotsDiff)
+
+    PLOTS_MODIFY_HELP = "Modify plot props associated with a target file."
```

Comment:
Can we call them "plot display properties" or at least "properties" in user-facing strings please? 🙂 

Lmk if so to change it in iterative/dvc.org/pull/1382 while it's in progress, to avoid requiring another docs PR for such change.

Reply:
@jorgeorpinel created https://github.com/iterative/dvc/pull/3983 .

Reply:
Thanks, updated docs PR too.